ChangePlacesOfMinMaxIn2Arrays/ChangePlacesOfMinMaxIn2Arrays/Program.cs

```diff
@@ -4,64 +4,108 @@ namespace ChangePlacesOfMinMaxIn2Arrays
-            }
-            Console.WriteLine("Array #{0}: [{1}]", 1, String.Join(", ", array1));
-            Console.WriteLine("Minimal array #{0} value: {1}\nMaximal array #{0} value: {2}", 1, minArray1Value, maxArray1Value);
-            
-            //Array #2 min/max values
-            for (int index = 0; index < array2.Length; index++)
+        /// <summary>
+        /// Get min array value
+        /// </summary>
+        /// <param name="array">Array</param>
+        /// <returns>minArrayValue</returns>
+        static int GetMinArrayValue(int[] array)
+        {
+            int minArrayValue = array[0];
+            for (int index = 1; index < array.Length; index++)
             {
-                minArray2Value = minArray2Value <= array2[index] ? minArray2Value : array2[index];
-                maxArray2Value = maxArray2Value >= array2[index] ? maxArray2Value : array2[index];
+                minArrayValue = minArrayValue <= array[index] ? minArrayValue : array[index];
             }
-            Console.WriteLine("Array #{0}: [{1}]", 2, String.Join(", ", array2));
-            Console.WriteLine("Minimal array #{0} value: {1}\nMaximal array #{0} value: {2}", 2, minArray2Value, maxArray2Value);
+            return minArrayValue;
+        }
 
-            //Array #1 - change min/max places
-            for (int index = 0; index < array1.Length; index++)
+        /// <summary>
+        /// Get max array value
+        /// </summary>
+        /// <param name="array">Array</param>
+        /// <returns>maxArrayValue</returns>
+        static int GetMaxArrayValue(int[] array)
+        {
+            int maxArrayValue = array[0];
+            for (int index = 1; index < array.Length; index++)
             {
-                //Change min to max
-                if (array1[index] == minArray1Value)
-                {
-                    array1[index] = maxArray1Value;
-                }
-                //Change max to min
-                else if (array1[index] == maxArray1Value)
-                {
-                    array1[index] = minArray1Value;
-                }
+                maxArrayValue = maxArrayValue >= array[index] ? maxArrayValue : array[index];
             }
-            Console.WriteLine("Changed min/max places in array #{0}: [{1}]:", 1, String.Join(", ", array1));
+            return maxArrayValue;
+        }
 
-            //Array #2 - change min/max places
-            for (int index = 0; index < array2.Length; index++)
+        /// <summary>
+        /// Get changed array with replaced min <-> max values
+        /// </summary>
+        /// <param name="array">Initial array</param>
+        /// <param name="minArrayValue">Minimal array value</param>
+        /// <param name="maxArrayValue">Maximal array value</param>
+        /// <returns>changedMinMaxArray</returns>
+        static int[] GetChangedMinMaxArray(int[] array, int minArrayValue, int maxArrayValue)
```

Comment:
Why you passing a value as a function parameter if you can use Min & MAx function for it?